mix.exs

```diff
@@ -5,16 +5,16 @@ defmodule Bamboo.Mixfile do
 
   def project do
     [app: :bamboo,
-     version: "0.4.2",
+     version: "0.5.0",
      elixir: "~> 1.2",
      source_url: @project_url,
      homepage_url: @project_url,
      compilers: compilers(Mix.env),
      test_coverage: [tool: ExCoveralls],
      preferred_cli_env: ["coveralls": :test, "coveralls.circle": :test],
      elixirc_paths: elixirc_paths(Mix.env),
-     description: "Straightforward, composable, and adapter based Elixir email library " <>
-     "for people that love piping. Adapters for Mandrill, Sendgrid, Sparkpost, in-memory, and test.",
+     description: "Straightforward, powerful, and adapter based Elixir email library." <>
+     " Works with Mandrill, Mailgun, Sendgrid, Sparkpost, in-memory, and test.",
```

Comment:
Actually, I will fix Sparkpost, but not SendGrid because the adapter is misspelled as `SendgridAdapter` in the code so this would be another breaking change that I don't want to make *quite* yet. Probably in the next version. I would want to make sure we add a nice error when we rename it to `SendGridAdapter` so people don't wonder why the adapter suddenly stopped working :)